outbound/index.js

```diff
@@ -158,6 +158,16 @@ exports.send_email = function () {
     }
 
     transaction.message_stream.add_line_end();
+
+    // Allow for the removal of Message-Id and/or Date headers which
+    // is useful when resending mail from a quarantine.
+    if (options && options.remove_msgid) {
```

Comment:
There's already several instances of `(options && options.something)`. I should think that at some point we'd do something proactive about it like `if (options === undefined) options = {}` or `const options = arguments[4] || {}` and thus simplify the following usages